app/src/main/java/org/oppia/app/shim/ViewBindingShim.kt

```diff
@@ -107,22 +107,29 @@ interface ViewBindingShim {
     attachToParent: Boolean
   ): View
 
+  /** Handles setting [DragDropInteractionItemsBinding]. */
   fun setDragDropInteractionItemsBinding(
     view: View
   )
 
+  /** Handles setting [DragDropInteractionItemsBinding]'s adapter. */
   fun setDragDropInteractionItemsBindingAdapter(
     adapter: RecyclerView.Adapter<RecyclerView.ViewHolder>
   )
 
+  /** Returns [DragDropInteractionItemsBinding]'s RecyclerView. */
   fun getDragDropInteractionItemsBindingRecyclerView(): RecyclerView
 
+  /** Returns [DragDropInteractionItemsBinding]'s dragDropContentGroupItem. */
   fun getDragDropInteractionItemsBindingGroupItem(): ImageButton
 
+  /** Returns [DragDropInteractionItemsBinding]'s dragDropContentUnlinkItems. */
   fun getDragDropInteractionItemsBindingUnlinkItems(): ImageButton
 
+  /** Returns [DragDropInteractionItemsBinding]'s dragDropAccessibleContainer. */
   fun getDragDropInteractionItemsBindingAccessibleContainer(): LinearLayout
 
+  /** Handles setting [DragDropInteractionItemsBinding]'s viewModel. */
```

Comment:
Here & elsewhere: ``view model`` not ``viewModel``. The current documentations seem to include variable names which is slightly confusing.

Reply:
Okay - I will try to make it clearer!